src/gluonts/mx/trainer/callback.py

```diff
@@ -338,3 +341,39 @@ def on_network_initializing_end(
         self, training_network: nn.HybridBlock
     ) -> None:
         copy_parameters(self.predictor.prediction_net, training_network)
+
+
+class TrainingTimeLimitCallback(Callback):
```

Comment:
I think we omitted the `callback` name from other callbacks.

```suggestion
class TrainingTimeLimit(Callback):
```